2019-04-24 14:14:01 [main] WARN  ClassPathXmlApplicationContext:554 - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanInitializationException: Could not load properties; nested exception is java.io.FileNotFoundException: class path resource [db.roperties] cannot be opened because it does not exist
2019-04-24 14:15:35 [main] WARN  ClassPathXmlApplicationContext:554 - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.CannotLoadBeanClassException: Cannot find class [org.apache.commons.dbcp2.BasicDataSource] for bean with name ' ataSource' defined in class path resource [applicationContext.xml]; nested exception is java.lang.ClassNotFoundException: org.apache.commons.dbcp2.BasicDataSource
2019-04-24 14:17:26 [main] WARN  ClassPathXmlApplicationContext:554 - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.CannotLoadBeanClassException: Cannot find class [org.apache.commons.dbcp2.BasicDataSource] for bean with name ' ataSource' defined in class path resource [applicationContext.xml]; nested exception is java.lang.ClassNotFoundException: org.apache.commons.dbcp2.BasicDataSource
2019-04-24 14:18:35 [main] WARN  ClassPathXmlApplicationContext:554 - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.CannotLoadBeanClassException: Cannot find class [org.apache.commons.dbcp2.BasicDataSource] for bean with name 'dataSource' defined in class path resource [applicationContext.xml]; nested exception is java.lang.ClassNotFoundException: org.apache.commons.dbcp2.BasicDataSource
2019-04-24 14:20:25 [main] WARN  ClassPathXmlApplicationContext:554 - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.CannotLoadBeanClassException: Cannot find class [org.apache.commons.dbcp2.BasicDataSource] for bean with name 'dataSource' defined in class path resource [applicationContext.xml]; nested exception is java.lang.ClassNotFoundException: org.apache.commons.dbcp2.BasicDataSource
2019-04-24 14:21:01 [main] WARN  ClassPathXmlApplicationContext:554 - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [applicationContext.xml]: Error setting property values; nested exception is org.springframework.beans.NotWritablePropertyException: Invalid property 'maxTotal' of bean class [org.apache.commons.dbcp.BasicDataSource]: Bean property 'maxTotal' is not writable or has an invalid setter method. Does the parameter type of the setter match the return type of the getter?
2019-04-24 15:27:43 [main] WARN  ClassPathXmlApplicationContext:554 - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanDefinitionStoreException: Invalid bean definition with name 'dataSource' defined in class path resource [applicationContext.xml]: Could not resolve placeholder 'jdbc.maxTotal' in value "${jdbc.maxTotal}"; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'jdbc.maxTotal' in value "${jdbc.maxTotal}"
2019-04-24 15:28:27 [main] WARN  ClassPathXmlApplicationContext:554 - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'transactionManager': Instantiation of bean failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.transaction.jta.JtaTransactionManager]: No default constructor found; nested exception is java.lang.NoClassDefFoundError: javax/transaction/UserTransaction
2019-04-24 15:31:09 [main] WARN  ClassPathXmlApplicationContext:554 - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'transactionManager': Instantiation of bean failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.transaction.jta.JtaTransactionManager]: No default constructor found; nested exception is java.lang.NoClassDefFoundError: javax/transaction/RollbackException
2019-04-24 15:32:09 [main] WARN  ClassPathXmlApplicationContext:554 - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanDefinitionStoreException: Invalid bean definition with name 'dataSource' defined in class path resource [applicationContext.xml]: Could not resolve placeholder 'jdbc.maxTotal' in value "${jdbc.maxTotal}"; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'jdbc.maxTotal' in value "${jdbc.maxTotal}"
